astro/src/components/Card.astro

```diff
@@ -0,0 +1,78 @@
+---
+import Carousal from './Carousel.astro';
+// export interface Props {
+// 	title: string;
+// 	body: string;
+// 	href: string;
+// }
+
+// const { href, title, body } = Astro.props;
+---
+
+<li class="link-card" style="align-items: center; display: flex; flex-direction: column; ; gap: 1rem;">
```

Comment:
Please use tailwind classes instead of inline css.
in this case it would be: `flex align-center flex-column gap1` or similar

We use Tailwind to have a standard way to define css